blockchain/index.test.js

```diff
@@ -1,5 +1,5 @@
 // Test suite for test-driven development
-const Blockchain = require('./blockchain');
+const Blockchain = require('./index');
```

Comment:
Previous file name is same with the current directory name, so currently the file name changed to `index.test.js` for JS convention